etna/analysis/outliers/density_outliers.py

```diff
@@ -52,7 +52,8 @@ def get_segment_density_outliers_indices(
 
     Returns
     -------
-    list of outliers' indices
+    List[int]
```

Comment:
`:` ?

Reply:
I wanted to make it the same in all outliers methods. Do we want to set `:` in all of them?